src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,48 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.bp-ColorPickerPalette {
+    position: absolute;
+    top: -55px;
+    right: 50%;
+    z-index: 1;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background: fade-out($black, .2);
```

Comment:
We should probably move this to a shared variable since it's used in ControlsBar, as well.

Reply:
Sounds good.